src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
```

Comment:
method hide() should invert hidden prop

f.e. if hidden was true, method hide() should turn in to false
if it was false, it should be true

Reply:
> method hide() should invert hidden prop
> 
> f.e. if hidden was true, method hide() should turn in to false if it was false, it should be true

Actually, it shouldn't toggle as the function is called `hide`
[FUNCTION NAMING] - if the function has a name hide this function should just hide but not toggle